content/learn/getting-started/node-with-typescript.pt.md

```diff
@@ -0,0 +1,129 @@
+---
+title: nodejs-with-typescript
+displayTitle: 'Node.js com a TypeScript'
+description: 'Descubra o porquê da TypeScript ser uma ferramenta impressionante e aprenda a usá-la sozinho.'
+authors: sbielenica, ovflowd, vaishnav-mk, nazarepiedady
+category: learn
+---
+
+## O Que é a TypeScript
+
+**[TypeScript](https://www.typescriptlang.org)** é uma linguagem de código-aberto na moda mantida e desenvolvida pela Microsoft. É amada e usada por muitos programadores de software em todo mundo.
+
+Basicamente, é um superconjunto de JavaScript que adiciona novas capacidades à linguagem. A adição mais notável é a definição de tipo estático, algo que não está presente na JavaScript simples. Graças aos tipos, é possível, por exemplo, declarar quais tipos de argumentos estamos esperando e o que é que é exatamente retornado em nossas funções ou qual é a forma exata do objeto que estamos a criar. A TypeScript é uma ferramenta realmente poderosa e abre um mundo novo de possibilidades nos projetos de JavaScript. Ela torna o nosso código mais seguro e robusto prevenindo muitos bugs antes mesmo do código ser moldado - ela captura problemas durante o desenvolvimento do código e se integra maravilhosamente com os editores de código como Visual Studio Code.
+
+Nós podemos falar a respeito de outros benefícios mais tarde, agora vamos ver alguns exemplos!
+
+### Exemplos
+
+Olhe este trecho de código e depois podemos desempacotá-lo juntos:
+
+```ts
+type User = {
+  name: string;
+  age: number;
+};
+
+function isAdult(user: User): boolean {
+  return user.age >= 18;
+}
+
+const justine: User = {
+  name: 'Justine',
+  age: 23,
+};
+
+const isJustineAnAdult: boolean = isAdult(justine);
+```
+
+A primeira parte (com a palavra-chave `type`) é responsável pela declaração do nosso tipo de objeto personalizado representando utilizadores. Depois utilizamos este tipo recentemente criado para criar uma função `isAdult` que aceita um argumento do tipo `User` e retorna um `boolean`. Depois disto, criamos o `justine`, o nosso dado de exemplo que pode ser usado para chamar a função definida anteriormente. Finalmente, criamos uma nova variável com a informação sobre se `justine` é um adulto.
```

Comment:
```suggestion
A primeira parte (com a palavra-chave `type`) é responsável pela declaração do nosso tipo de objeto personalizado representando utilizadores. Depois utilizamos este tipo recentemente criado para criar uma função `isAdult` que aceita um argumento do tipo `User` e retorna um `boolean`. Depois disto, criamos o `justine`, o nosso dado de exemplo que pode ser usado para chamar a função definida anteriormente. Finalmente, criamos uma variável com a informação sobre se `justine` é um adulto.
```

Reply:
Portuguese has its own way to describe things. and using the structure and norms of English is not legal.

I do not see the necessity of changing it, and since there are no grammar or syntactical errors, it should remain as it is.

Reply:
The word "criamos" (create) already refers to something new.
"Criamos uma nova" (create a new) there is a pleonasm problem